### DISCUSSION TITLE
Better feedback for ava test

### DISCUSSION THREAD

**Discussion by conartist6** [avajs/ava#2891] [Ideas]
For a while now I've been writing `ava test somefile.js`. As I also use `ava debug` I assumed that this tool followed the subcommand pattern. Instead what I now realize is that there is no `test` subcommand -- I've just been filtering test based on two patterns, one that exists and one that does not. I propose that if the users types `ava test` and no tests match the pattern, a warning is displayed which suggests just running `ava`.

**Comment by novemberborn** [comment]
Interesting. AVA 4 supports directory patterns, and does print a warning if it didn't match any tests. So `ava test` would try to run tests inside the `test/` directory. Could you give this a go with `npm i -D -E ava@next` and see if the message is sufficient?
